pkg/controller/blockdevice/blockdevice_controller.go

```diff
@@ -91,6 +92,24 @@ func (r *ReconcileBlockDevice) Reconcile(request reconcile.Request) (reconcile.R
 		return reconcile.Result{}, err
 	}
 
+	switch instance.Status.ClaimState {
+	case openebsv1alpha1.BlockDeviceReleased:
+		jobController := cleaner.NewJobController(r.client, request.Namespace)
+		cleanupTracker := &cleaner.CleanupStatusTracker{JobController: jobController}
+		bdCleaner := cleaner.NewCleaner(r.client, request.Namespace, cleanupTracker)
+		ok, err := bdCleaner.Clean(instance)
```

Comment:
Is this a blocking call? Or will this just schedule a cleaner job.

Reply:
I see it is not - though a new cleaner is created every time, there can be only one job per bd that is in progress. the Clean function will return false - while the cleaner job is in progress, and a true once it is completed. 

Probably a comment here explaining this help.